models/models.js

```diff
@@ -36,6 +36,19 @@ exports.fetchAllArticles = ()=>{
             return rows
         })
 }
+
+exports.fetchCommentsByArticleId = (article_id)=>{
+    return db.query(`SELECT * FROM comments
+        WHERE article_id = $1
+        ORDER BY created_at DESC`, [article_id])
+        .then(({rows})=>{
+            if(rows.length === 0){
```

Comment:
What if my article_id is valid, but my article no comments? This would cause this promise to be rejected, and with an inaccurate error message